packages/teams-js/test/public/pages.spec.ts

```diff
@@ -210,42 +335,917 @@ describe('AppSDK-TeamsAPIs', () => {
+          'The library has not yet been initialized',
+        );
+      });
+
+      Object.values(FrameContexts).forEach(context => {
+        it(`pages.tabs.getMruTabInstances should successfully getTabInstance when no parameters are passed and initialized with ${context} context`, async () => {
+          await utils.initializeWithContext(context);
+          const promise = pages.tabs.getMruTabInstances();
+          let message = utils.findMessageByFunc('getMruTabInstances');
+
+          utils.respondToMessage(message, expectedTabInstanceParameters);
+          expect(message).not.toBeNull();
+          expect(promise).resolves.toBe(expectedTabInstanceParameters);
+        });
+
+        it(`pages.tabs.getMruTabInstances should be undefined getTabInstance when parameters are passed and  initialized with ${context} context`, async () => {
+          await utils.initializeWithContext(context);
+          const promise = pages.tabs.getMruTabInstances(expectedTabInstanceParameters);
+          let message = utils.findMessageByFunc('getMruTabInstances');
+
+          utils.respondToMessage(message, expectedTabInstanceParameters);
+          expect(message).not.toBeNull();
+          expect(promise).resolves.toBe(expectedTabInstanceParameters);
+        });
+
+        it(`pages.tabs.getMruTabInstances should be undefined when no parameters are passed and initialized with ${context} context`, async () => {
+          await utils.initializeWithContext(context);
+          const promise = pages.tabs.getMruTabInstances();
+          let message = utils.findMessageByFunc('getMruTabInstances');
+
+          utils.respondToMessage(message);
+          expect(message).not.toBeNull();
+          expect(promise).resolves.toBeUndefined();
+        });
+      });
+    });
+
+    describe('Testing pages.tabs.isSupported function', () => {
+      it('pages.tabs.isSupported should return false if the runtime says pages.tabs is not supported', () => {
+        utils.setRuntimeConfig({ apiVersion: 1, supports: { pages: {} } });
+        expect(pages.tabs.isSupported()).not.toBeTruthy();
+      });
+
+      it('pages.tabs.isSupported should return true if the runtime says pages.tabs is supported', () => {
+        utils.setRuntimeConfig({
+          apiVersion: 1,
+          supports: { pages: { tabs: {} } },
+        });
+        expect(pages.tabs.isSupported()).toBeTruthy();
+      });
+    });
+  });
+
+  describe('Testing pages.config namespace', () => {
+    describe('Testing pages.config.initialize function', () => {
+      it('pages.config.initialize should successfully register settings.save and settings.remove handler', async () => {
+        pages.config.initialize();
+        const messageForSettingeSaveHandler = utils.findMessageByFunc('settings.save');
+        const messageForSettingeRemoveHandler = utils.findMessageByFunc('settings.remove');
+        expect(messageForSettingeSaveHandler).toBeNull();
```

Comment:
I'm a bit confused about how this validates that pages.config.initialize registered the settings.save and settings.remove handlers.

Reply:
For example, if I comment out `pages.config.initialize` and run this test, it still passes.